src/store/modules/swap/ethBancor.ts

```diff
@@ -2352,6 +2352,10 @@ export class EthBancorModule
     this.setTolerance(tolerance);
   }
 
+  get isCurrentMainnet() {
```

Comment:
Perhaps ```isSupportedNetwork``` instead.

Reply:
Changed it. and warning appears only if it is not the mainnet.